ui/packages/catalog-ui-search/src/main/webapp/js/CQLUtils.js

```diff
@@ -9,254 +9,295 @@
  * <http://www.gnu.org/licenses/lgpl.html>.
  *
  **/
-/*global define*/
 /*jshint bitwise: false*/
-define([
-    'jquery',
-    'js/cql',
-    'js/DistanceUtils.js',
-    'component/singletons/metacard-definitions'
-], function ($, cql, DistanceUtils, metacardDefinitions) {
+const $ = require('jquery');
+const cql = require('js/cql');
+const DistanceUtils = require('js/DistanceUtils.js')
 
-    return {
-        sanitizeForCql: function (text) {
-            return text.split('[').join('(').split(']').join(')').split("'").join('').split('"').join('');
-        },
+function sanitizeForCql(text) {
+    return text.split('[').join('(').split(']').join(')').split("'").join('').split('"').join('');
+}
 
-        sanitizeGeometryCql: function (cqlString) {
-            //sanitize polygons
-            let polygons = cqlString.match(/'POLYGON\(\((-?[0-9]*.?[0-9]* -?[0-9]*.?[0-9]*,?)*\)\)'/g);
-            if (polygons) {
-                polygons.forEach((polygon) => {
-                    cqlString = cqlString.replace(polygon, polygon.replace(/'/g, ''));
-                });
-            }
+function generateAnyGeoFilter(property, model) {
+    switch (model.type) {
+        case 'LINE':
+            return {
+                type: 'DWITHIN',
+                property: property,
+                value: 'LINESTRING' +
+                sanitizeForCql(JSON.stringify(lineToCQLLIne(model.line))),
```

Comment:
Is there a reason for the capital I in LIne?
Does this not get wrapped in `(` and `)` like the others?

Reply:
Wow, good eye! The capital I is fixed. This doesn't need the extra parens because `LINESTRING`s only have one set and `lineToCQLLine()` returns an array which gets stringified and the square brackets are converted into parens by `sanitizeForCql()`